front/lib/api/auth_wrappers.ts

```diff
@@ -237,10 +237,22 @@ export function withPublicAPIAuthentication<T, U extends boolean>(
           });
         }
 
-        const auth = await Authenticator.fromAuth0Token({
+        const authRes = await Authenticator.fromAuth0Token({
           token: decoded.value,
           wId,
         });
+        if (authRes.isErr()) {
+          return apiError(req, res, {
+            status_code: 403,
+            api_error: {
+              type: "not_authenticated",
+              message:
+                "The user does not have an active session or is not authenticated.",
+            },
+          });
+        }
+        const auth = authRes.value;
+
         if (auth.user() === null) {
```

Comment:
This cannot happen anymore with your change since you return directly "user_not_found" if there's no user. 🙏🏻

Reply:
Yeah until that code changes. auth_wrappers wants a user so let's keep the verification :+1: